specification/monitor/resource-manager/Microsoft.Insights/stable/2018-03-01/metricAlert_API.json

```diff
@@ -797,7 +797,8 @@
             "Average",
             "Minimum",
             "Maximum",
-            "Total"
+            "Total",
+            "Count"
```

Comment:
Adding new value in enum allowed values is a breaking change to SDK. Suggest to change this in new API version.

Reply:
@raych1 
Thanks for the review.

It is a bug fix. There is a misalignment between metric alerts RP and metric alerts swagger.
Currently Metric alerts RP returns all the 5 values while swagger documented only 4 values.

As the RP already supports the new property value, can we push it to the current API version?
Any other suggestion how to fix it?

Reply:
This enum is modeled as a string so it shouldn't affect the SDK. For the sake of this comment, someone from the SDK team should ack on this.

Reply:
It's ok to align with RP behavior in this case.